src/web/ballot.js

```diff
@@ -11,6 +11,8 @@ import { getAddrFromEth } from '../vendor/rnode-client-js/src/rev-address';
 import { transferMulti_rho } from '../rho/transfer-multi';
 import { lookup_rho } from '../rho/lookup';
 
+const voteruri = "rho:id:1ri71weozwuoanef9zit5p7ooafkmkzhkwo6phgaourcknbmi6ke7t";
```

Comment:
This isn't really const, is it? It will change before we're done. Please open an issue to address this in a later PR.

Reply:
https://github.com/rchain-community/rv2020/issues/98